### PR TITLE
javascript-forms - Alan Ho

### PR DESCRIPTION
**Pull request by alanho-05**
https://user-images.githubusercontent.com/23649469/230165017-c21089ca-e04d-482a-85d3-90efcc1af19d.mov

### CODE REVIEW

File: javascript-forms/form-values.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const $form = document.querySelector('form');
+const input = {};
+
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  input['name:'] = $form.elements.name.value;
+  input['email:'] = $form.elements.email.value;
+  input['message:'] = $form.elements.message.value;
```
Comment (shawnkost) [inline javascript-forms/form-values.js:8]: You're adding an extra colon by doing this. If you check out the console log, the object is showing `name::`. Either remove the colon, or use dot notation

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change requested!

File: javascript-forms/form-values.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  console.log('messageData:', input);
+  this.reset();
```
[the remark this answers is not in the document]
Comment (alanho-05) [reply javascript-forms/form-values.js]: Visual studio code added the `this` automatically when I wrote `reset()`.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**